### ISSUE TITLE
Implement the new crown assets in Frontend (v5.x)

### ISSUE THREAD

**Issue by querkmachine** [alphagov/govuk-frontend#4178]
## What

Implement the new crown assets in GOV.UK Frontend v5.x.

## Why

To ensure that teams are using the new GOV.UK branding on any new services.

## Who needs to work on this

- Developers

## Who needs to review this

- Developers
- Designers
- Accessibility specialist

## Done when

```[tasklist]
- [x] Inline SVG graphic has been implemented in Header component
- [x] Sass has been updated to account for the new lockup (link hover styles, product name integration, etc.)
- [x] Favicon and web sharing graphics have been replaced
- [x] A feature flag to activate the new header graphic has been implemented
```

**Comment by stevenjmesser** [comment]
Merging and releasing this is dependent on https://github.com/alphagov/govuk-frontend/issues/4059.

**Comment by querkmachine** [comment]
Shifting this into blocked for the time being as this might become work for a 5.x release instead, with a transitional piece being merged into 5.0 instead.
